src/rp2_common/tinyusb/CMakeLists.txt

```diff
@@ -104,8 +104,46 @@ if (EXISTS ${PICO_TINYUSB_PATH}/${TINYUSB_TEST_PATH})
 
     add_library(tinyusb_board INTERFACE)
     target_sources(tinyusb_board INTERFACE
-            ${PICO_TINYUSB_PATH}/hw/bsp/raspberry_pi_pico/board_raspberry_pi_pico.c
+            ${PICO_TINYUSB_PATH}/hw/bsp/rp2040/family.c
     )
+    target_include_directories(tinyusb_board INTERFACE ${PICO_TINYUSB_PATH}/hw/bsp/rp2040/boards/sdk_selected)
 
+    set(PICO_TINYUSB_PATH_INTERNAL ${PICO_TINYUSB_PATH} CACHE INTERNAL "")
+    add_library(tinyusb_example_support INTERFACE)
+    target_compile_definitions(tinyusb_example_support INTERFACE
+            CFG_TUSB_OS=OPT_OS_PICO
+            BOARD=pants
```

Comment:
:jeans:

Reply:
oops - i was checking something by deliberately using an invalid board ;-) that ain't gonna compile

Reply:
fixed